cogboard-app/src/main/kotlin/com/cognifide/cogboard/CogboardConstants.kt

```diff
@@ -47,6 +48,7 @@ class CogboardConstants {
 
         const val EVENT_SEND_MESSAGE_TO_WEBSOCKET = "cogboard.websocket.message"
         const val EVENT_SAVE_BOARDS_CONFIG = "cogboard.config.boards.save"
+        const val EVENT_CONTENT_UPDATE_WIDGET_CONFIG = "cogboard.config.widget.contentUpdate"
```

Comment:
i think better name would be `EVENT_UPDATE_WIDGET_CONTENT `

Reply:
Agreed, fixed.